src/hooks/settingsHooks/useGetSettings.ts

```diff
@@ -116,12 +117,34 @@ const convertFromBe = (backendSettings: BackendSiteSettings): SiteSettings => {
   }
 }
 
+const extractPassword = (
+  passwordData: BackendPasswordSettings
+): string | null => {
+  const { isAmplifySite } = passwordData
+  if (!isAmplifySite) return null
+
+  return passwordData.password
+}
+
 export const useGetSettings = (
   siteName: string
 ): UseQueryResult<SiteSettings> => {
   return useQuery<SiteSettings>(
     [SETTINGS_CONTENT_KEY, siteName],
-    () => SettingsService.get({ siteName }).then(convertFromBe),
+    async () => {
+      const siteSettings = await SettingsService.get({ siteName })
+      const passwordSettings = await SettingsService.getPassword({ siteName })
+      const convertedSettings = convertFromBe({
```

Comment:
why can't we do convertFromBe(siteSettings)?

Reply:
Oops we can now, changed in https://github.com/isomerpages/isomercms-frontend/pull/1316/commits/fd7ce4c772e35f616905cf5b54e49f773857b8a4